app/helpers.go

```diff
@@ -0,0 +1,31 @@
+package app
+
+import (
+	"GOBootcamp/app/models"
+	"encoding/json"
+	"log"
+	"net/http"
+)
+
+func mapPostToJSON(p *models.Posts) models.JsonPost {
```

Comment:
AS it contain the same values and properties, do we really require to have this conversion? even the json mapping is the same